app/build.gradle

```diff
@@ -29,6 +29,9 @@ dependencies {
     implementation "com.android.support:design:$rootProject.supportLibraryVersion"
     implementation "com.android.support.constraint:constraint-layout:$rootProject.constraintLayoutVersion"
     // Test
+    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
```

Comment:
Please stick to advices given in early PRs. I've requested to use version tags from rootproject and you're not following that. See in here you've added the same library twice. I expect quality code as you've been in the project for some time.

Reply:
@CloudyPadmal actually sir while Rebuilding/Cleaning the project Android studio automatically acheived this. As far as I believe I did not touch the build gradle file. But sir, I am definitely looking into and correcting it. Thanks.